jax/_src/api.py

```diff
@@ -1906,6 +1907,17 @@ def lower(*args, **kwargs) -> stages.Lowered:
     """
     _experimental_lowering_platform = kwargs.pop(
         '_experimental_lowering_platform', None)
+    _experimental_override_lowering_rules = kwargs.pop(
```

Comment:
Add a TODO to remove these deprecated parameters?

These are not used in OSS, so maybe you can just kill them in the next CL? I think you will have to change partir which uses _experimental_override_lowering_rules to use the new thing.

Reply:
Indeed, I will clean this in a separate change.
Added TODO